src/constants.py

```diff
@@ -167,3 +167,12 @@
 PRODUCT_DOCS_PERSIST_DIR = "./vector-db/ocp-product-docs"
 SUMMARY_INDEX = "summary"
 SUMMARY_DOCS_PERSIST_DIR = "./vector-db/summary-docs"
+
+# cache constants
+IN_MEMORY_CACHE = "in-memory"
+IN_MEMORY_CACHE_MAX_SIZE = 1000
```

Comment:
```suggestion
IN_MEMORY_CACHE_MAX_ENTRIES = 1000
```

makes it clearer what the units are

Reply:
Changed as suggested.